friends/src/components/FriendsList/Forms.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+
+
+//styles
+import {Button, Form, FormGroup, Input, } from 'reactstrap';
+
+
+
+
+class Forms extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {  
+            newName: '',
+            newAge: '',
+            newEmail: ''
+        }
+    }
+
+  //changeHandler // gets moved to form component
+  changeHandler = event => {
+    this.setState({
+        friend: {
+        [event.target.name]: event.target.value
+    }   
```

Comment:
You need to set Form state here, look carefully at the structure of your state and see how you can update them accordingly based on the corresponding input's value change.